classes/action_functions.py

```diff
@@ -76,41 +78,51 @@ class Write_txt():
     def directory(self):
         if os.path.exists(f'''{self.file_path}/{self.month}'''):
             '''Se o diretório já existir.'''
             self.file_path = f'''{self.file_path}/{self.month}'''
+            print(self.file_path)
             return (True, 'Diretório já existente')
         else: 
             '''Se o diretório não existir ele cria o diretório'''
-            os.makedirs(f'''{self.file_path}/{month}''')
+            os.makedirs(f'''{self.file_path}/{self.month}''')
             self.file_path = f'''{self.file_path}/{self.month}'''
+            print(self.file_path)
+
             return (True, 'Diretório criado')
 
     '''Escreve / cria arquivo txt no diretório especificado.'''
-    def write(string):
-        if os.path.exists(f'''{self.file_path}/{datetime.now().strftime('%d-%m-%Y')}.txt'''):
+    def write(self, txt):
+        filess = self.file_path
+        files = filess[1]
+        if os.path.exists(f'''{files}/{datetime.now().strftime('%d-%m-%Y')}.txt'''):
             '''Se arquivo txt existe.'''
-            with open(f'''{self.file_path}/{datetime.now().strftime('%d-%m-%Y')}.txt''', 'r') as file:
+            with open(f'''{files}/{datetime.now().strftime('%d-%m-%Y')}.txt''', 'r') as file:
                 '''Abre o arquivo em modo de leitura.'''
                 if (file.read()[len(file.read())-2:]) == '\n':
                     '''Verifica se o último valor é uma quebra de linha.'''
-                    with open(f'''{self.file_path}/{datetime.now().strftime('%d-%m-%Y')}.txt''', 'a') as text:
+                    with open(f'''{files}/{datetime.now().strftime('%d-%m-%Y')}.txt''', 'a') as text:
                         '''Abre novamente o arquivo mas em modo de adicionar texto.'''
-                        text.write(string) # Escreve no arquivo txt.        
+                        text.write(txt) # Escreve no arquivo txt.        
                 else:
                     '''Se não for, ele inicia a linha com uma quebra de linha.'''
-                    with open(f'''{self.file_path}/{datetime.now().strftime('%d-%m-%Y')}.txt''', 'a') as text:
-                        text.write('\n' + string) # Escreve no arquivo txt.          
+                    with open(f'''{files}/{datetime.now().strftime('%d-%m-%Y')}.txt''', 'a') as text:
+                        text.write('\n' + txt) # Escreve no arquivo txt.          
         else:
             '''Se não existe, arquivo txt é criado.'''
-            with open(f'''{self.file_path}/{datetime.now().strftime('%d-%m-%Y')}.txt''', 'a') as file:
+            with open(f'''{files}/{datetime.now().strftime('%d-%m-%Y')}.txt''', 'a') as file:
                 '''Abre o arquivo em modo de adicionar text'''
-                file.write(string) # Escreve no arquivo txt.
\ No newline at end of file
+                file.write(txt) # Escreve no arquivo txt.
+
+if __name__ == '__main__':
+    save = Where_save().find()
+    print(save)
+    salve = Write_txt().write("boa noite")
```

Comment:
```suggestion
    salve = Write_txt().write("boa noite")
    
```
Enter final do arquivo (boas práticas)